tests/gpaw/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# add tests for GPAW4QMCPACK converter
```

Comment:
is `tests/gpaw` intended to test the converter or workflow?
if converter, move under `tests/converters/gpaw`
if workfload, rename as `gpaw4qmcpack`

Reply:
The present test is only for the converter, not workflow, and no running with GPAW. I wasn't sure where to put it. Your proposition sounds good, I'll make the change.